sdk/src/loom/System.ls

```diff
@@ -0,0 +1,131 @@
+package loom
+{
+    import system.platform.Platform;
+    import system.platform.PlatformType;
+    
+    public class System
+    {
+        private var _system:natSystem;
+        
+        /**
+         * Constructor
+         * 
+         * The constructor takes an optional ID, that can be used to distinguish multiple system objects
+         * using the special cmdWithId delegate.
+         * 
+         * @param id The Id to be given to this system object. Defaults to -1.
+         */
+        public function System(id:int = -1)
+        {
+            _system = new natSystem(id);
+        }
+        
+        /**
+         * Run the provided command and get feedback from that command. The feedback will be returned using the onData
+         * delegate. Data is automatically extracted from both STDOUT and STDERR.
+         * 
+         * NOTE: If you wish to run a synchronous multi-part command, it is reccomended that you use the multCmd function because of
+         * platform specific differences in command syntax.
+         * 
+         * @param command The command that will be run through the native command interface.
+         * 
+         * @see multCmd
+         */
+        public function cmd(command:String):void { _system.cmd(command + " 2>&1"); }
```

Comment:
Is the 2>&1 idiom portable?

Should this be called run or something? Look at the Adobe AIR API for this.